### DISCUSSION TITLE
[13/10]Which smartphone do you prefer?

### DISCUSSION THREAD

**Discussion by fujiwaraka** [NAIST-SE/AttractingContributors#912] [2023 Attraction Contributions]
Could you add your favorite smartphone to my list?

https://github.com/fujiwaraka/Which-smartphone-do-you-prefer-

あなたの好きなスマートフォンをリストに追加してくれませんか？

**Comment by rfLENtlr** [comment]
I like Oppo Reno Smartphone.
Please contribute to my repository as well!!
https://github.com/rfLENtlr/AttractingContributors-OS

**Reply by fujiwaraka** [reply]
You haven't made a pull request.
Could you please make a pull request?

**Reply by rfLENtlr** [reply]
I just sent a pull request.

**Reply by fujiwaraka** [reply]
Thank you so much.

**Comment by bconfessor** [comment]
I like Xiaomi, cheap and useful! 
I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Reply by fujiwaraka** [reply]
Thanks.OK
